plugins/source/oracle/codegen/generate_OracleClients_struct.py

```diff
@@ -0,0 +1,166 @@
+# Outputs an `OracleClients` struct, and intialization code for the struct.
+# To be used with one of `--imports`, `--initialization`, or `--struct` flags.
```

Comment:
I'm not sure what work this is doing exactly, but we may be able to adapt https://github.com/cloudquery/plugin-sdk/pull/590 to also fit this use case? Seems like it should be possible

Reply:
Or adapt this - https://github.com/cloudquery/cloudquery/tree/main/plugins/source/azure/codegen0. Def don't see a strong need for python as Go has very strong library for parsing source code which is done in azure. maybe we can outsource this utility to a seperate codegen library as it's not really part of the SDK

Reply:
This creates the `oracle_clients.go` file, that is used by the later stages of codegen for discovery purposes.
I kinda went for the fastest possible way (so I used the `https://github.com/ungerik/pkgreflect` library, and this script that does regex on it's output).

I would be happy to not rewrite this (because this was kind of a one-off - the OracleClients struct doesn't really need to be regenerated).

But I can rewrite this in golang if oyu both feel strongly about htis.

Reply:
Honestly, I'd prefer to just delete this script rather than rewrite it more generically right now, since I am not sure rewriting will bring any benefit (the `OracleClients` struct will remain the same)..

Reply:
Yeah don't worry, I don't think it's a blocker. Deleting the script and updating the file by hand from now on sounds like a good option for now.

Reply:
As per slack discussion:
- removed `// Code generated by codegen` lines
- changed `.github` presubmits to only test doc-changes, not code-changes.
- Removed the python script: you can find it at https://gist.github.com/shimonp21/70c0a3cceaeab3f054e76c2846810b3e